common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/Indexer.scala

```diff
@@ -97,14 +107,20 @@ object Indexable extends Logging {
 
       def version(work: Work[Identified]) =
         work match {
-          case Work.Visible(_, _, state) if state.nMergedSources >= 10 =>
-            throw new RuntimeException(
-              s"Work ${work.state.canonicalId} has ${work.state.nMergedSources} >= 10 merged sources; versioning/ingest may be inconsistent")
+          case Work.Visible(_, _, state)
+              if state.nSources >= versionMultiplier =>
+            warnOnTooManySources(work)
+            // Clamp the increment so as not to override higher versions
+            versionMultiplier - 1
```

Comment:
Do you not want to multiply by the version? This is consistently going to return `version = 999`, no matter what the source version is.

Reply:
oops - yes!